ddtrace/contrib/kafka/patch.py

```diff
@@ -33,6 +33,14 @@
 
 
 class TracedProducer(confluent_kafka.Producer):
+    def __init__(self, config, *args, **kwargs):
+        super(TracedProducer, self).__init__(config, *args, **kwargs)
+        self._dd_bootstrap_servers = (
```

Comment:
nit: can we set the kafkax.HOST_LIST tag here instead of adding the `_dd_bootstrap_servers` attribute?

Reply:
Done via `pin`.